src/pages/detail-page/subpages/components/ContactBlock.tsx

```diff
@@ -19,7 +19,7 @@ const ContactBlock: React.FC<ContactBlockProps> = ({ title, contacts }) => {
           key={index}
           title={contact.organization + suffix}
           isContactFragment
-          email={contact.emails[0]}
+          email={contact.emails ? contact.emails[0] : undefined}
```

Comment:
no big deal, but what if `contact.emails[0] = ""`?

Reply:
If any details is falsy("", undefined, null, etc...), the corresponding area will show something like "no *** data". Steve has this design, but it needed to change according to their discussion. So I will implement all the falsy data when the design is approved. I will then raise a ticket for it (will paste the link here shortly)

Reply:
https://github.com/aodn/backlog/issues/5780